website/examples/e-commerce/App.js

```diff
@@ -4,212 +4,275 @@ import {
-}
-
-const Hit = ({ item }) => {
-  const icons = [];
-  for (let i = 0; i < 5; i++) {
-    const suffixClassName = i >= item.rating ? '--empty' : '';
-    const suffixXlink = i >= item.rating ? 'Empty' : '';
-
-    icons.push(
-      <svg
-        key={i}
-        className={`ais-RatingMenu-starIcon ais-RatingMenu-starIcon${suffixClassName}`}
-        aria-hidden="true"
-        width="24"
-        height="24"
-      >
-        <use xlinkHref={`#ais-RatingMenu-star${suffixXlink}Symbol`} />
-      </svg>
-    );
-  }
-  return (
-    <article className="hit">
-      <div className="product-picture-wrapper">
-        <div className="product-picture">
-          <img
-            alt={item.name}
-            src={`https://res.cloudinary.com/hilnmyskv/image/fetch/h_300,q_100,f_auto/${item.image}`}
+          <HitsPerPage
+            items={[
+              {
+                label: '16 hits per page',
+                value: 16,
+              },
+              {
+                label: '32 hits per page',
+                value: 32,
+              },
+              {
+                label: '64 hits per page',
+                value: 64,
+              },
+            ]}
+            defaultRefinement={16}
           />
-        </div>
-      </div>
-      <div className="product-desc-wrapper">
-        <div className="product-name">
-          <Highlight attribute="name" hit={item} />
-        </div>
-        <div className="product-type">
-          <Highlight attribute="type" hit={item} />
-        </div>
-        <div className="product-footer">
-          <div className="ais-RatingMenu-link">{icons}</div>
-          <div className="product-price">${item.price}</div>
-        </div>
-      </div>
-    </article>
-  );
-};
-
-const CustomResults = connectStateResults(({ searchState, searchResult }) => {
-  if (searchResult && searchResult.nbHits === 0) {
-    return (
-      <div className="results-wrapper">
-        <div className="no-results">
-          No results found matching{' '}
-          <span className="query">{searchState.query}</span>
-        </div>
-      </div>
-    );
-  } else {
-    return (
-      <div className="results-wrapper">
-        <section id="results-topbar">
-          <div className="sort-by">
-            <label>Sort by</label>
-            <SortBy
-              items={[
-                { value: 'instant_search', label: 'Featured' },
-                { value: 'instant_search_price_asc', label: 'Price asc.' },
-                { value: 'instant_search_price_desc', label: 'Price desc.' },
-              ]}
-              defaultRefinement="instant_search"
-            />
-          </div>
-          <Stats />
-        </section>
-        <ConnectedHits />
-        <footer>
-          <Pagination showLast={true} />
-        </footer>
-      </div>
-    );
-  }
-});
+        </header>
 
-const ConnectedSearchBox = connectSearchBox(CustomSearchBox);
-const ConnectedHits = connectHits(CustomHits);
+        <Hits
```

Comment:
We should use the `ScrollTo` component to reset the scroll position to the top of the list.